doc/source/notebooks/advanced/multioutput.pct.py

```diff
@@ -168,7 +168,9 @@ def plot_model(m, lower=-8.0, upper=8.0):
 
 # %%
 # create multi-output kernel
-kernel = gpf.kernels.SharedIndependent(gpf.kernels.RBF(1) + gpf.kernels.Linear(1), output_dim=P)
+kernel = gpf.kernels.SharedIndependent(
```

Comment:
we used to have two non ARD kernels, now we have one ARD (the RBF one) and one non-ARD (the linear one). Is that what you intended to do?

Reply:
Ah, somehow I misremembered and assumed the default had been ARD=True. I'm happy to take out the `lengthscales` argument - not that it matters in this concrete example as D=1 ...